src/contribute/contribution-guide.md

```diff
@@ -301,6 +301,10 @@ From another local branch, run:
 	$ git branch -d <my-branch>
 	$ git push <GitHub_user> --delete <my-branch>
 
+### Stale pull requests
+
+The community will close stale pull requests in order to keep the project healthy. A pull request becomes stale after its author fails to respond to actionable comments for 60 days.  Author of a closed pull request is welcome to reopen the same pull request again in the future. The associated JIRAs will be unassigned from the authors but will stay open.
```

Comment:
`will be unassigned from the authors` -> `will be unassigned from the author`

Reply:
Done.